### PR TITLE
Create Meeting Notes 51

### PR DESCRIPTION
**Pull request by AntMeyer1**

### CODE REVIEW

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:49]:
```suggestion
**Guests**: Christophe Garant, Michael Liesenfelt, Andrew Westberg, Alessandro Konrad, Thomas Vellekoop
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:117]:
```suggestion
**Michael** -: The notes. I was looking at a couple of those and there was that I believe it's CIP 37, this one, 45 and I guess 50, that is all with pledge and the centralization metrics is concerned. It's very, well, it's hard to go through you, you'd expect so. [inaudible 00:07:03] so I don't know, I wish I could give us [inaudible 00:07:04] and maybe there's a way that we can bundle them, simpler path to a review. I don't remember it's particularities on the whole.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:79]:
```suggestion
**Robert Phair** -: As far as using the equations in CIPs and in GitHub content, the difficulty they ran into with another CIP 50, did they ever work out the presentation of LaTeX type equations on the mobile platform? Someone said they weren't working.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:87]:
```suggestion
**Robert Phair** -: I get it. I hadn't thought of it that way. They could go back to the original form if they want to see that kind of detail in the equations.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:95]:
```suggestion
**Christoff** -: Oh, Hey everybody. Yeah. So as I learned in CIP 50, yeah, the mobile definitely doesn't render, but then there's some brackets, especially curly brackets that doesn't render on GitHub even on desktop. So that's why we reverted back to basically screenshot equations from external source. Pictures.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:81]:
```suggestion
**Matthias** -: So there is only Christophe on the chat commenting that. Yeah, they have done equations on GitHub, prone to some issues as you learn from CIP 50. So you are saying it's specifically on a mobile device, that it is not supported yet?
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:83]:
```suggestion
**Robert Phair** -: That's what I think Christophe was saying.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:89]:
```suggestion
**Matthias** -: Probably, it looks good in VS Code than Jupyter Notebooks, said Christophe. But then in GitHub and stuff, can't render, I guess, does not render on mobile in brackets, in desktop. I can't speak if, yeah, not, actually, since you are here, it'll faster than just writing on the chat. Can we just unmute you. Yes, it's done.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:91]:
```suggestion
**Christophe** -: Can you guys hear me?
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:95]:
```suggestion
**Christophe** -: Oh, Hey everybody. Yeah. So as I learned in C50, yeah, the mobile definitely doesn't render, but then there's some brackets, especially curly brackets that doesn't render on GitHub even on desktop. So that's why we reverted back to basically screenshot equations from external source. Pictures.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:99]:
```suggestion
**Christophe** -: No it's specifically GitHub and I did some digging and I don't know, I guess the layman's interpretation, it was something with like Java... Sorry, I just got a call. JavaScript rendering on the website itself with GitHub. And there were some conflicts and it seems like a mess, but it'll look good in VS Code and even Jupyter Notebooks and just regular [inaudible 00:04:25] it's fine. But when it's in GitHub README, it breaks. So you have to add some extra back slashes and curly brackets and yeah, we eventually just reverted back. For simpler equations, it is probably okay. But for more complex things, it breaks it seems.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:103]:
```suggestion
**Christophe** -: Yeah. If it works great, but yeah, just be prepared to check.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:107]:
```suggestion
**Christophe** -: I personally haven't reviewed it, but now I will. I wasn't that aware of it so...
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:111]:
```suggestion
**Christophe** -: But I'll do that.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:197]:
```suggestion
**Matthias** -: Christophe is also unmuted. If you also maybe have some info on that. So he rejoined and we still can't hear you. Nope. Well, that's unfortunate too, I'm not seeing any Michael. Michael is in chat. You don't see him? He's in the list.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:201]:
```suggestion
**Matthias** -: I'm failing on audio. Yeah, we can see that. So I don't know if Christophe maybe wants to give us that shorter bit. I think you've both chatting on that. Okay. Or we can leave it for next time or has it written a update maybe? On the meeting windows app. Yeah.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:203]:
```suggestion
**Christophe** -: I guess I, can you guys hear me?
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:209]:
```suggestion
**Christophe** -: Thanks, Michael can't speak, but I guess I just put on, I just updated a pull request on the rank leverage based ranking recommendation last night, which we've removed a grading system to kind of be a little bit more objective. So it's just kind of a 10 is the best and then zero is the worst kind of recommendation, but again, it's just a recommendation, not actually core to it.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:135]:
```suggestion
**Robert Phair** -: You saw when you, at the [inaudible 00:10:43] when you looked over that there was a suggestion that the MAC addresses on the machine might be considered relevant to the point of CIP 45. And I've suggested that maybe that should be moved off to the Cardano forum and kept out of this CIP 45 discussion on GitHub. I believe it's irrelevant to the CIP as stated and, I think the rationale for doing that is not correct.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:139]:
```suggestion
**Robert Phair** -: Anything regarding the MAC addresses and how the MAC address could be used to further the ambitions of that CIP I think can be skipped over in considering the CIP. It's been a digression.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:121]:
```suggestion
**Matthias** -: I think this is a bit what Michael is also saying in the chats, by saying I've included CIP 45 in the CIP 50 references since the first draft. And I think it's good to keep that as part of this, the discussions, because in a way it's part of the whole same discussions. And as we are now trying to get more and more attention from IOG research, I mean, we've got not going to have that discussion five time, one for each of those CIPs, but probably once and for all, at least for the next proposal to come and see which direction we take.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:165]:
```suggestion
**Matthias** -: And this is something I would love to see in the rationale section as well. Exactly what you just explained. This is a collaborative effort between actual people building with, in the music industry, on, kind of, i don't know, the blockchain in general and that's also why we justifying some of the choices that have been made.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:179]:
```suggestion
**Andrew** -: Yeah. We've had several two-hour meetings with people. There were 20 different groups in the room, either producing music on Cardano or representing wallets or players or things like that. So yeah, I will add to the rationale section before the next meeting that this comes up.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:343]:
```suggestion
**Matthias** -: And this is something I would love to see in the rationale section as well. Exactly what you just explained. This is a collaborative effort between actual people building with, in the music industry, on, kind of, i don't know, the blockchain in general and that's also why we justifying some of the choices that have been made.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:356]:
```suggestion
**Andrew** -: Yeah. We've had several two hour meetings with people. There was 20 different groups in the room, either producing music on Cardano or representing wallets or players or things like that. So yeah, I will add to the rationale section before next meeting that this comes up.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (rphair) [inline BiweeklyMeetings/2022-08-16.md:125]:
```suggestion
**Matthias** -: Yeah, so perhaps that's also kind of, I don't know, part of CIP 50 rationale section, they have a kind of alternatives considered and list a bit of the pro and cons of those various other CIPs and why maybe what was taken from them, what was rejected from them for which reason, because it might actually also have discussions and having one place to centralize all the results there might help too.
```

**Review by rphair — COMMENTED** [review]
I think it's important to transcribe our guest's name correctly after all the contribution he's made to CIP-0050 😎 https://twitter.com/TheStophe

There is another **really major problem here**: a huge section of duplicated text.  I can't mark it below because it would get lost in the flood of small mistakes (I didn't try to correct everything, mostly just the proper names & references).

This section begins with the CIP-0050 anchor ("So, okay. Let's keep it on the agenda for review for the next meeting. And since we have 10 minutes left") and is nearly identical with the same passage which currently shows after the CIP-0060 anchor as well.  The duplicated text is slightly differently transcribed and it's too much for me to try to fix this without the original recording.  I would suggest playing it again in chronological order to know which redundant section to delete. 🧐

Once the little edits are processed I'll give another read & approve once we have a text flow that's as much as possible like we actually had a the meeting... even with the missing audio sections 😎

File: BiweeklyMeetings/2022-08-16.md
Comment (KtorZ) [inline BiweeklyMeetings/2022-08-16.md:373]:
```suggestion
So the content itself is okay technically-wise, but there are quite a few shortcomings when it comes to how it is written. So we asked the author to fix that. CIP 30, we have agreed to just close up requests. It's been inactive for a while. The author is not responsive. I can't close it myself, unfortunately, because I'm on windows. But if any of the other auditors want to do it, please do. CIP 45 then the using pledge as a bidding program, there is some overlap with CIP 50. This is part of the broader discussions about rewards and incentives and there were concerns being raised about this actually incentivizing large pledge owners to just create more stake pools without actually impacting the decentralization of Cardano. At least not in a good way.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (KtorZ) [inline BiweeklyMeetings/2022-08-16.md:374]:
```suggestion
Then we went through CIP 68, which we agreed to put that last check for next week and merge as proposed given that's, the technical content was, already has been through many iterations although there were a few points raised regarding this two-step beta data identifications that could perhaps, they discussed as well. So anyway, that will be a full two weeks before we go to the last check and we'll see for further updates after that. We've been through the NFT identity proposal to CIP 68, which also sounds technically okay, in the context of CIP 25, although it narrates the same shortcomings and flaws. So perhaps further discussions are needed. The format of that one is also a bit fleshed out and could deserve well, that be a bit more love and especially some more details on, well, the actual details of the CIPs. Possibly also some renaming, since this seems to be heavily inspired by the W3C recommendations, maybe could be made a bit clearer from the CIP title and finally CIP 60, the token metadata extension for music-related tokens, which comes as an extension of CIP 25 and has been a collaboration between many different individuals in the music industry around Cardano.
```

File: BiweeklyMeetings/2022-08-16.md
Comment (KtorZ) [inline BiweeklyMeetings/2022-08-16.md:375]:
```suggestion
It is mainly proposing a set of 20+ new fields that are specific to music-related NFTs. And we agreed to discuss it as the last check for not the next biweekly, but one after which will be also in the US-friendly time zone. And that's it, sorry, we are four minutes beyond the clock and thank you all.
```

**Review by KtorZ — APPROVED** [review]

**Review by rphair — COMMENTED** [review]
I can't approve this for merge since the huge section of duplicated text pointed out in https://github.com/cardano-foundation/CIPs/pull/320#pullrequestreview-1084753802 is still there.  Currently this can be seen by opening these two links in separate browser tabs & going back & forth between them:

- https://github.com/AntMeyer1/CIPs/blob/Create-Meeting-Notes-52/BiweeklyMeetings/2022-08-16.md#pr307
- https://github.com/AntMeyer1/CIPs/blob/Create-Meeting-Notes-52/BiweeklyMeetings/2022-08-16.md#pr242

Maybe worse, there are some difference between the sections, since at this point they are still edited slightly differently & probably not straightforward to merge if one of them was deleted.

I will try to submit a revised version of the document (as Matthias just did in the last editing round?) since a GitHub edit would be too difficult to select.  Internet access has been unsteady here but I should have time to do this in the next couple of days if nobody else does.

**Review by rphair — APPROVED** [review]
OK I think I finally got it.  The removed duplicate section had to be the correct one in time since only one matches the flow of the conversation... to double check, this can be seen in the diff for the last commit.

Let's please never get a transcription this bad from those folks again  because I can't measure how much time has been lost here by the duplicated section leading the meeting (and all attempts to edit its minutes) around in an endless loop 🤪 ... also adjusted anchors to match the flow of the dialogue which jumped between CIPs a couple of times.

**Pull request merged**
